docs/tutorials/additional-resources/community.md

```diff
@@ -25,6 +25,10 @@ Below is a list of additional resources from the community.
 
 - [Using Trivy Kubernetes in OVHCloud documentation.](https://docs.ovh.com/gb/en/kubernetes/installing-trivy/)
 
+## Trivy AWS
```

Comment:
It is not AWS scanning. They just run Trivy on EC2.

Reply:
Oh, will update it